pkgs/development/python-modules/biliass/default.nix

```diff
@@ -7,14 +7,18 @@
 
 buildPythonPackage rec {
   pname = "biliass";
-  version = "1.3.5";
-  disabled = pythonOlder "3.6";
+  version = "1.3.7";
+  disabled = pythonOlder "3.7";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "sha256-kgoQUX2l5YENEozcnfluwvcAO1ZSxlfHPVIa9ABW6IU=";
+    sha256 = "sha256-P7K3bt8MXD6HoJ7duQ9lG99yj0lVwn9SqEEC/TUudK4=";
   };
 
+  postPatch = ''
+    sed -i -e 's/4.21.9/4.21.8/' setup.py
```

Comment:
Please add more context what is being substituted

Reply:
```suggestion
    # relax protobuf version constraint
    sed -i -e 's/4.21.9/4.21.8/' setup.py
```

Is this OK?